src/pages/home/ReportScreen.tsx

```diff
@@ -719,7 +719,11 @@ function ReportScreen({route, currentReportID = '', navigation}: ReportScreenPro
         // After creating the task report then navigating to task detail we don't have any report actions and the last read time is empty so We need to update the initial last read time when opening the task report detail.
         Report.readNewestAction(report.reportID);
     }, [report]);
-    const firstReportAction = reportActions[0];
+    const mostRecentReportAction = reportActions[0];
+    const shouldShowMostRecentReportAction =
+        !!mostRecentReportAction &&
+        !ReportActionsUtils.isActionOfType(mostRecentReportAction, CONST.REPORT.ACTIONS.TYPE.CREATED) &&
+        !ReportActionsUtils.isDeletedAction(mostRecentReportAction);
```

Comment:
This flag doesn't check for one-expense/invoice reports and makes the preview to be displayed. As such, https://github.com/Expensify/App/issues/49853 resulted.